pkg/cli/environment.go

```diff
@@ -89,6 +90,10 @@ type EnvSettings struct {
 	BurstLimit int
 	// QPS is queries per second which may be used to avoid throttling.
 	QPS float32
+	// ThreeWayMergeForUnstructured controls whether to use three way merge
+	// patch for unstructured objects (custom resource, custom definitions,
+	// etc).
+	ThreeWayMergeForUnstructured bool
```

Comment:
About the actions, it currently is implemented as a global option. The alternative to that is to add it to the upgrade and install action commands individually. The environment is just an additional option. But I have no issue removing it from there.

Did I understand correctly that there should not be an environment variable to configure this flag and that that the option should not be global but implemented for the install and upgrade actions individually instead?

Reply:
Well, technically adding the option to the install action should have no effect unless `TakeOwnership` is also part of the CLI and not only of the SDK. But since the SDK needs it anyway for all resources of a chart that are already present and are supposed to be adopted by having the  `TakeOwnership` option set, it might make sense to add it, too. Or even go as far as to enable `TakeOwnership` for install and upgrade commands in the CLI, too?

Reply:
Adoption of resources owned by helm is a feature of helm that is independent of `TakeOwnership`, so it will make sense to have it in the CLI in any case.